apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/EndpointJsonComparatorBase.java

```diff
@@ -58,8 +62,8 @@ public void assertJsonDiffIsEmpty(String endpointId, String endpointResponse) {
         ServerTestUtils.readFile(
             ServerTestUtils.generatePathForEndpointJsonFile(
                 getExpectedJsonResponseDir(), endpointId));
-    AssertUtils.assertJsonEquals(
-        expectedJson, endpointResponse, ServerTestUtils.JSON_COMPARE_IGNORED_PATHS);
+    String replacedResponse = replaceIgnoredFieldsWithFillerText(endpointId, endpointResponse);
```

Comment:
So this turned into a whole thing. We apparently have two different methods of redacting fields in the JSON response - one utilizes the `replaceIgnoredFieldsWithFillerText` which looks through the entire document and replaces any node with `IGNORED_FIELD` if it matches the given key name and regex.

The second method uses a dynamically constructed regex to mimc something like a JSONPath query and replaces anything it finds with `--ignored-value--`. The change from a numeric value to a UUID in all of the internal ID fields did not play nicely with the regex-based approach since these IDs appeared in a lot of different subtypes.

I went ahead and replaced all usages of the second approach for the adjudicated endpoints with the first approach. Ideally we could just rip out all of the code for the second approach entirely since it's redundant, but the PAC endpoints still use it and refactoring those would be a larger effort.